### PR TITLE
HCAP-522 Seed Scripts

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/HCAP-522

- Makefile commands for seeding
- Participant seed script, completely different implementation but similar functionality to `feed-sites.js`
- Update docs with new functionality
- Fix feed-sites where it hangs on successful completion

### CODE REVIEW

File: server/scripts/feed-participants.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const path = require('path');
+const fs = require('fs');
+const { dbClient } = require('../db');
+
+const { parseAndSaveParticipants } = require('../services/participants');
+
+(async () => {
+  if (require.main === module) {
```
Comment (dbayly-freshworks) [inline server/scripts/feed-participants.js:10]: Neat!

File: server/scripts/feed-participants.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    await dbClient.connect();
+    console.log('Successfully Connected to DB');
+
+    try {
+      fs.readFile(path.resolve(__dirname, `xlsx/${process.argv[2]}`), async (err, data) => {
```
Comment (dbayly-freshworks) [inline server/scripts/feed-participants.js:15]: We should probably verify that argv[2] exists just to be safe.

**Review by dbayly-freshworks — COMMENTED** [review]
2 small comments, but otherwise this looks good.

File: server/scripts/feed-participants.js (hunk 1 above)
Comment (arranfw) [reply server/scripts/feed-participants.js:10]: copy and pasted from `feed-sites` 😄

File: README.md [hunk 3]
```diff
@@ -160,12 +169,15 @@ and then submits to the /form endpoint. Example of file names:
 - Health Career Access Program - Expression of Interest - 8353ec90e6ea8727.xml
 - Health Career Access Program - Expression of Interest - 91113c10e0xx872x (1).xml
 
-#### `npm run feed-sites my_spreadsheet.xlsx`
+#### `npm run feed-sites my_spreadsheet.xlsx` and  `npm run feed-participants my_spreadsheet.xlsx`
```
Comment (ashtonmeuser) [inline README.md:172]: Of all people @arranfw 
```suggestion
#### `npm run feed-sites my_spreadsheet.xlsx` and `npm run feed-participants my_spreadsheet.xlsx`
```

**Review by ashtonmeuser — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (arranfw) [reply README.md:172]: shame

**Review by Zenen-freshworks — APPROVED** [review]
looks good, seems like the previous comments covered everything

**Review by dbayly-freshworks — APPROVED** [review]

**Pull request merged**